ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -183,6 +199,7 @@ def create_msg_regex(self, file_lsit):
                                        skipinitialspace=True)
 
                 for index, row in enumerate(csvreader):
+                    row = [item for item in row if item != ""]
                     self.print_diagnostic_message('[diagnostic]:processing row:%d' % index)
                     self.print_diagnostic_message('row:%s'% row)
                     try:
```

Comment:
There is present general exception handler. It will also process IndexError.

Reply:
Why do you want the test fail with an exception when everything is good except the regex file had an empty line which resulted in IndexError? I would suggest to filter out the empty lines so that we don't hit that condition or fix the error message to access only within the allowed range.

Reply:
Current code skips empty and commented lines:
```
#-- Ignore commented Lines and Empty Lines
if (not row or row[0].startswith(comment_key)):
    self.print_diagnostic_message('[diagnostic]:skipping row[0]:%s' % row[0])
    continue
```

Reply:
Will be fixed.

Reply:
Done.